### DISCUSSION TITLE
OneAdmin card component

### DISCUSSION THREAD

**Discussion by pamelahu** [twilio-labs/paste#2972] [Office hours] [answered]
I'm working on OneAdmin (a.k.a. the "redesign" of Monkey) and specifically on the "Account overview" page!

You can see the framework/bones/IA of OneAdmin that we have so far here: https://www.figma.com/file/2xLMo0ang2t6KxVj65RRA5/(PH)-Account-Overview%3A-Hi-fi-Mocks?node-id=274%3A137629&t=bPOtYVsSsN1lgy0h-1 

Brittany Hamtil has been helping out with template-izing the account overview page: https://www.figma.com/file/DG0LmndffdCWl1kFJtC0UM/1A-Templates?node-id=201%3A135258&t=znNMsxHzubeyGy4J-1 

Now we're wondering how to approach this new card component:
![Screenshot 2023-01-23 at 7 12 38 PM](https://user-images.githubusercontent.com/18605252/214180893-a32f5956-d424-4a3d-a7d5-cba3ce0db02e.png)

Is it an entirely new component? Do we just customize the content that goes in the card?

And just FYI: Kendal Kirby is leading this OneAdmin initiative and we've been in touch with Liberty on creating a custom theme.

**Comment by loreina** [comment]
Thanks for the write-up @pamelahu! Our team will follow up here after office hours. :)

**Comment by gloriliale** [comment, marked as answer]
Hi @pamelahu! Just wanted to share the office hours recording with you in case you'd like to revisit any of our conversations. [Here's the video]( https://www.loom.com/share/cb51374d80e646cd991cef69bf2cd9db), and the password is posted in the #help-design-system channel. Thanks again for coming to office hours!

**Comment by pamelahu** [comment]
@gloriliale Thanks for the recording! I had a quick follow up question for the description item component, can lists be placed side-by-side since a single column takes up a lot of space?
![Screenshot 2023-01-27 at 6 14 44 PM](https://user-images.githubusercontent.com/18605252/215223495-5771f987-4234-4c27-8a2e-d374c19d2e45.png)

**Reply by gloriliale** [reply]
Hi @pamelahu! They can be side by side- I'd recommend using referencing the new [spacing/layout guidelines](https://paste.twilio.design/foundations/spacing-and-layout) for some details on our recommendations!
